trussworks/react-uswds
# feat: Update GovBanner to match USWDS 2.8.0 release

# Summary

## Related Issues or PRs

closes #456 

## How To Test

Verify copy changes in Storybook for .gov and .mil versions of the banner for both English and Spanish

### Screenshots (optional)
Storybook stories:
![image](https://user-images.githubusercontent.com/15805554/104259699-ed944980-544f-11eb-8744-a3a23923bb24.png)

defaults to "english" & ".gov":
![image](https://user-images.githubusercontent.com/15805554/104259754-fedd5600-544f-11eb-86b2-7061b5219c8b.png)

with "english" and ".gov" passed:
![image](https://user-images.githubusercontent.com/15805554/104259777-0f8dcc00-5450-11eb-987b-e3da8f7d25de.png)

with "english" and ".mil" passed:
![image](https://user-images.githubusercontent.com/15805554/104259803-23393280-5450-11eb-8884-826f2e68c118.png)

with "spanish" and ".gov" passed:
![image](https://user-images.githubusercontent.com/15805554/104259836-38ae5c80-5450-11eb-95c8-6b7c88dcb66c.png)

with "spanish" and ".mil" passed:
![image](https://user-images.githubusercontent.com/15805554/104259851-4237c480-5450-11eb-9bbd-93063ebcb2bf.png)
